bundles/io/org.eclipse.smarthome.io.rest.core/src/main/java/org/eclipse/smarthome/io/rest/core/persistence/ServiceBean.java

```diff
@@ -17,5 +17,13 @@
     ServiceBean() {
     }
 
+    /**
+     * Service Name
+     */
     public String name;
+
+    /**
+     * Persistence service class
+     */
+    public String classname;
```

Comment:
A classname is imho not appropriate to be exposed to the client side.
We should rather have a field "type", which could be "Standard", "Queryable" and "Modifiable" (or even only 0,1,2 if this is nothing that we ever would want to display directly to the user).

Btw, I think it would be a good time to "upgrade" PersistenceServices to ESH conventions wrt exposing them to UIs - we usually have "Ids" for services (which would be the current "name" field) and a label, which can be localized and returns the label that should be used in UIs (see e.g. [here](https://github.com/kaikreuzer/smarthome/blob/310c041a5309b28970948387bec66e2353ef0629/bundles/core/org.eclipse.smarthome.core.voice/src/main/java/org/eclipse/smarthome/core/voice/TTSService.java), where I have upgraded the TTSService interface in a similar way.